Token_Contracts/contracts/StandardToken.sol

```diff
@@ -22,7 +22,7 @@ contract StandardToken is Token {
             balances[_to] += _value;
             Transfer(msg.sender, _to, _value);
             return true;
-        } else { return false; }
+        } else { revert(); }
```

Comment:
Could now use `require(balances[msg.sender] >= _value);` and then `balances[msg.sender] -= _value; ...`  (May as well update comment too if going to use require.).   Applies to next change below too, line 37.